onnxruntime/core/providers/cpu/tensor/split.cc

```diff
@@ -81,6 +83,8 @@ Status Split::Compute(OpKernelContext* context) const {
     status = ComputeImpl<int32_t>(*context, input);
   else if (input.IsDataTypeString())
     status = ComputeImpl<std::string>(*context, input);
+  else if (input.IsDataType<int64_t>())
```

Comment:
nit: Maybe move this `if` check before 'string' and after 'int32' ? Most likely, this op will be used for numeric tensors and can potentially save the if check (although the check itself is pretty cheap)...